src/aat/resources/features/F-064/F-064.feature

```diff
@@ -0,0 +1,54 @@
+@F-064
+Feature: F-064: Retrieve workbasket input details for dynamic display
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-217
+  Scenario: should retrieve trigger when the case and event exists
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And it is submitted to call the [retrieve workbasket input details for dynamic display] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [code is HTTP-200 OK]
+    And the response has all other details as expected
+
+  @S-218
```

Comment:
Scenario mismatch with excel:

S-218 | CCD   Data Store Api :: Retrieve workbasket input details for dynamic display | 403:   Forbidden | must   return 403 when request provides authentic credentials without authorised   access to the operation
-- | -- | -- | --